dashboards/google-blockchain-node-engine/blockchain_node_engine_dashboard.json

```
@@ -0,0 +1,257 @@
+{
+  "displayName": "Blockchain Node Engine Dashboard",
```

Small nit to remove redundant "Dashboard" from the name here.

```suggestion
  "displayName": "Blockchain Node Engine",
```